packages/test/test-end-to-end-tests/src/test/opReentrancy.spec.ts

```diff
@@ -38,6 +39,13 @@ describeCompat(
 		const testContainerConfig: ITestContainerConfig = {
 			fluidDataObjectType: DataObjectFactoryType.Test,
 			registry,
+			runtimeOptions: {
+				compressionOptions: {
+					// Compression is always disabled for these tests
+					minimumBatchSizeInBytes: Number.POSITIVE_INFINITY,
+					compressionAlgorithm: CompressionAlgorithms.lz4,
+				},
```

Comment:
Are you sure this is still needed?  Try removing it, I think it should work.

If it does turn out to be necessary, once we sort out why, switch to this:


```suggestion
				compressionOptions: disabledCompressionConfig,
```

Reply:
applied changes